governator-core/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'java'
 
 dependencies {
     compile     project(':governator-api')
+    compile     "javax.annotation:javax.annotation-api:1.3.2"
```

Comment:
This project is no longer maintained as it is part of the EE stuff that was transitioned to Eclipse:

https://github.com/javaee/javax.annotation/blob/master/README.md

The version of this package released by eclipse is:

https://mvnrepository.com/artifact/jakarta.annotation/jakarta.annotation-api/1.3.5